api/src/main/java/org/openmrs/module/metadatadeploy/bundle/CoreConstructors.java

```diff
@@ -56,6 +60,16 @@
 public class CoreConstructors {
 
 	/**
+	 * Constructs a dbsessionFactory source
+	 * @return the transient object
+	 */
+	public static DbSessionFactory dbSessionFactory(SessionFactory sessionFactory) {
```

Comment:
Why did you put DbSessionFactory in CoreConstructors? Shouldn't it be in the data access layer?